doc/analytics.md

```diff
@@ -28,14 +28,20 @@ endpoints or rake tasks.
 
 Here are some example queries to pull out particular insights.
 
-### How many subscribers does a list have at a particular point in time
+### How many subscribers does a list have
 
 ```ruby
+Subscription.active.where(subscriber_list_id: 4194).count
+```
+
+### How many subscribers did a list have at a particular point in time
+
+```ruby
+date = "2018-6-01"
 Subscription.where(
-    "created_at > ? AND (ended_at IS NULL OR ended_at <= ?)",
-    "2018-06-01",
-    "2018-07-01"
-  ).where(subscriber_list_id: 4194).count
+  "created_at < ? AND (ended_at IS NULL OR ended_at <= ?)",
```

Comment:
I'm a bit confused here.  Does `ended_at` mean when they unsubscribed?  If so shouldn't this query be `created_at < ? AND (ended_at IS NULL OR ended_at > ?)`?

Might also be worth using named interpolations to make it clearer you're using the same date, at which point you wouldn't need the extra variable either:
```
User.where(["name = :name and email = :email", { name: "Joe", email: "joe@example.com" }])
# SELECT * FROM users WHERE name = 'Joe' AND email = 'joe@example.com';
```

Could we also turn this into a method/scope on a SubscriberList instance to further reduce confusion?

Reply:
Yes, I think it needs changing as I'm only getting a very small number of subscribers when I run it

Reply:
Oh you're right, it should be `>`. I'll look at creating a scope as well.

Reply:
@vanitabarrett @elliotcm Thanks for the review! I've pushed up a fix with a scope, what do you think?